api/src/main/java/ca/bc/gov/educ/api/edx/repository/EdxUserRepository.java

```diff
@@ -11,4 +14,11 @@
 public interface EdxUserRepository extends JpaRepository<EdxUserEntity, UUID>, EdxUserRepositoryCustom {
 
   List<EdxUserEntity> findEdxUserEntitiesByDigitalIdentityID(UUID digitalIdentityID);
+
+  @Modifying
```

Comment:
Delete EdxUserSchool method should be in EdxUserSchoolRepository

Reply:
Hi @sukanya-rath, I have moved the method as you suggested.